build.sbt

```diff
@@ -96,6 +99,13 @@ lazy val commonSettings = Seq(
 
   scalafmtOnCompile := true,
 
+  // We can't use Scalafix in Scala 3 yet.
+  libraryDependencies ++= forScalaVersions {
+    case (2, _) => List(compilerPlugin(scalafixSemanticdb))
+    case _ => List.empty
+  }.value,
+  scalafixOnCompile := forScalaVersions { case (2, _) => true; case _ => false }.value,
```

Comment:
I'm a bit wary of having more and more build stuff conditionally disabled or modified for Scala 3.0... let's keep close track of these and remove them as soon as Scala 3.0 gets support for it.